Tests/UnitTests/Fatras/Digitization/ChannelMergerTests.cpp

```diff
@@ -0,0 +1,96 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <boost/test/unit_test.hpp>
+
+#include "ActsFatras/Digitization/ChannelMerger.hpp"
+#include "ActsFatras/Digitization/DigitizationError.hpp"
+
+#include <Acts/Tests/CommonHelpers/FloatComparisons.hpp>
+#include <Acts/Utilities/Definitions.hpp>
+#include <Acts/Utilities/Helpers.hpp>
+#include <Acts/Utilities/ParameterDefinitions.hpp>
+
+namespace ActsFatras {
+
+BOOST_AUTO_TEST_SUITE(Digitization)
```

Comment:
```suggestion
BOOST_AUTO_TEST_SUITE(FatrasDigitization)
```

I am not a big fan of putting the tests into the `ActsFatras` namespace. They are not part of the library proper; they are users of the library. To avoid having to add `ActsFatras::` everywhere, `using ActsFatras;` should be fine. To organize tests, Boost already provides the `BOOST_AUTO_TEST_SUITE` macro that you are already using. I think this will place everything in a separate namespace.

Reply:
Ok, namespace removed.